topic_monitor/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup
+
+setup(
+    name='topic_monitor',
+    version='0.0.0',
+    packages=[],
+    py_modules=[
+        'launch_files.launch_depth_demo',
+        'launch_files.launch_durability_demo',
+        'launch_files.launch_fragmentation_demo',
+        'launch_files.launch_reliability_demo',
+        'scripts.data_publisher',
+        'scripts.topic_monitor',
```

Comment:
@dirk-thomas they are installed as `console_script` `entry_point`s at https://github.com/ros2/demos/pull/90/files/cc986ede15e919cc03d8091fe1233d235470ee36#diff-02cee11aa71cd20c15b15b26cc0dfa9bR29 . it is also necessary to install the modules at this line (which is what the other python packages do)
I have tested that they launch fine on all platforms